docs/roadmap.md

```diff
@@ -9,16 +9,14 @@ The goal of this document is to give you a hint on what's next. However, since a
 ![](./assets/img/roadmap.png)
 
 ## In tech preview
-
-This categories means the technology is here, but not officially released for production usage at the moment.
+_Technology that is here, but not officially released for production usage._
```

Comment:
I'd like to get a space after a title, even if it's not visible in the HTML result. Find it more readable when editing the MD file.